test/TestProjects/FirstTest-TypeSpec/FirstTest-TypeSpec.tsp

```diff
@@ -367,3 +367,17 @@ namespace ProtocolAndConvenient {
 @head
 @convenientAPI(true)
 op headAsBoolean(@path id: string): void;
+
+
```

Comment:
Add these to cadl-ranch.

Reply:
hello @pshao25 add test cases in pr https://github.com/Azure/cadl-ranch/pull/428. After this is merged, we will onboard the test cases and remove those.